docs/user-guide/topics/metrics_store.md

```diff
@@ -0,0 +1,54 @@
+---
+template: main.html
+---
+
+# Metrics store
+
+One of Iter8's key advantages is that it incorporates its own metrics store simplifying the set up and execution of A/B/n and performance tests. This metrics store, based on BadgerDB is not suitable for production use. It is suitable only for a single instance of Iter8.
+
+To support production use cases, the metrics store can be replaced with another distributed database such as Redis. Currently Iter8 supports:
+
+- BadgerDB
+- Redis
+
+See [below](#contribute-a-new-metrics-store-implementation) for details on how to contribute additional implementations.
+
+## Using Redis as the metrics store 
+
+We assume that Redis is deployed. For example, for a basic deployment:
+
+```shell
+kubectl create deploy redis --image=redis/redis-stack:latest --port=6379
+kubectl expose deploy redis --port=6379
+```
+
+Run Iter8 identifying the metrics store implementation as `redis` and identify the endpoint:
+
+```shell
+helm upgrade --install --repo https://iter8-tools.github.io/iter8 --version 0.18 iter8 controller  \
+--set clusterScoped=true \
+--set metrics.implementation=redis \
+--set metrics.redis.addresss=redis:6379
+```
+
+## Contribute a new metrics store implementation
+
+To contribute a new metrics store implementation:
+
+1. Create an [issue](https://github.com/iter8-tools/iter8/issues) for discussion.
+
+2. Submit a pull request on the [Iter8 project](https://github.com/iter8-tools/iter8) with the following updates:
+
+    - Create sub-folder in [storage](https://github.com/iter8-tools/iter8/tree/master/storage) and provide an implementation of this [interface](https://github.com/iter8-tools/iter8/blob/master/storage/interface.go) including test cases.
```

Comment:
I recall that the BadgerDB and redis implementations are both named `simple.go`. This was just an artifact of when we were first toying around with the idea of using a database. I think if we are going to make users aware that they can contribute, we should also change this file name to a more sensible name.